luigi_tools/parameter.py

```diff
@@ -156,13 +161,17 @@ def _is_sequence(data):
     return isinstance(data, collections.abc.Sequence) and not isinstance(data, str)
 
 
+def _is_optional(cls):
+    return typing_extensions.get_origin(cls) is typing.Union and type(None) in typing.get_args(cls)
```

Comment:
Why don't you need to test for `typing.Optional`?

Reply:
Because `typing.Optional[X]` is not `typing.Optional` and the origin of optional is a Union that can also be other things so I have to check that NoneType is in there to make sure it's the optional type.

Reply:
Ah ok
I love typing :laughing:

Reply:
Yeah, it's brilliant..